### PR TITLE
Feature/forms

### PR DESCRIPTION
**Pull request by ReAliens**
in this pull request, I implemented the following:-
- Create forms to perform the following functions:
  - Creates a Post on behalf of the current_user.
  - Create a comment on behalf of the current_user.
 - Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
```
Comment (hiromataba) [inline app/controllers/comments_controller.rb:1]: - [ ] Please add a new action to render the form for creating a comment

File: config/routes.rb [hunk 2]
```diff
@@ -4,4 +4,8 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
   end
+  get "/posts/new", to: "posts#new"
+  post "/posts/create", to: "posts#create"
+  post "/users/:user_id/posts/:id/like", to: "likes#create"
+  post "/users/:user_id/posts/:id/create_comment", to: "comments#create"
```
Comment (hiromataba) [inline config/routes.rb:10]: - [ ] Please add a route to render the form to create a comment

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @ReAliens      :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Nice commit history :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please add information in your  README file on how to run your project locally 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ReAliens** [comment]
thanks for your feedback 
but I founded the bug was that I rendered the partial in wrong place because route is already exist

**Review by andersonlebon — APPROVED** [review]
## Hi @ReAliens 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [x] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**